src/main/java/org/jvnet/hudson/test/recipes/WithTimeout.java

```diff
@@ -38,12 +37,14 @@
  *
  * All the tests time out after some number of seconds by default, but this recipe
  * allows you to override that value.
+ * 
+ * No need for @JenkinsRecipe in JUnit 4 as it's implemented directly in {@link JenkinsRule} 
+ * by the private method: getTestTimeoutOverride.
```

Comment:
Consider referencing the public `JenkinsRule.timeout method()` as what a user can call, instead of referencing a private method here in public documentation.

Reply:
Good point. Actually the comment was not meant to be read by "user" but more "developer", to prevent future myself to re-do the same useless change :) So I move the comment outside of the JavaDoc.